wpeverest/everest-forms
# Feature - Form migrator for wpforms

### All Submissions:


### Changes proposed in this Pull Request:

This PR introduces a new feature called **Form Migrator** which helps to import the form and entry of **WPForms** into our forms easily with one click.

### How to test the changes in this Pull Request:

1. Install and activate the WPForms.
2. Create some forms with free fields.
3. Go to Everest form > tools menu > form migrator tab > Select wpforms
4. verify the list of forms
5. Try to import single and multiple forms at once.
6. After importing the form, try to import entries of that form and verify the entries.
7. Also Verify by importing the contact form 7 because some codes are modified.

### Types of changes:


### Other information:


### Changelog entry

> Feature - Form migrator for wpforms.
